.github/workflows/test-lang-java.yml

```diff
@@ -62,9 +62,13 @@ jobs:
           java-version: ${{ matrix.java }}
 
       - name: Lint
+        env:
+          GRADLE_ENTERPRISE_ACCESS_KEY: ${{ secrets.GE_ACCESS_TOKEN }}
```

Comment:
Since this is the full java test CI I wonder if we should make the env config global (only in this file)

Reply:
Makes sense, I made this change 👍🏻